### DISCUSSION TITLE
Live audio input available?

### DISCUSSION THREAD

**Discussion by bsrega** [stride3d/stride#1437] [Q&A] [answered]
I'm just getting started with Stride.
In a project, every available audio input-device seem so be a file based, 
but is there a way to select a live-audio device?
Thanks, Stephan

**Comment by manio143** [comment, marked as answer]
At the moment there is not built-in support for that. I think the `DynamicSoundSource` abstract class could be inherited to support some live audio interface, but those are usually platform specific. Can you provide an example of a use case?

**Comment by bsrega** [comment]
Use it in performances, e.g. the voice triggers something in a scene, the guitar something else..
